packages/sit-onyx/src/components/OnyxTooltip/OnyxTooltip.vue

```diff
@@ -78,20 +105,45 @@ const ariaPattern = shallowRef(createPattern());
 watch(type, () => (ariaPattern.value = createPattern()));
 
 const tooltip = computed(() => ariaPattern.value?.elements.tooltip);
-const trigger = computed(() => toValue<HTMLAttributes>(ariaPattern.value?.elements.trigger));
+const trigger = computed(() => toValue<object>(ariaPattern.value?.elements.trigger));
+
+const tooltipWrapperRef = ref<HTMLElement>();
+const tooltipRef = ref<HTMLElement>();
+const { openDirection, updateOpenDirection } = useOpenDirection(tooltipWrapperRef);
+const { wedgePosition, updateWedgePosition } = useWedgePosition(tooltipWrapperRef, tooltipRef);
+
+// update open direction on resize to ensure the tooltip is always visible
+onMounted(() => {
+  const updateOnResize = () => {
+    updateOpenDirection();
+    updateWedgePosition();
+  };
+
+  window.addEventListener("resize", updateOnResize);
+
+  // initial update
+  updateOpenDirection();
+  updateWedgePosition();
+});
+
+onBeforeUnmount(() => {
+  window.removeEventListener("resize", updateOpenDirection);
+  window.removeEventListener("resize", updateWedgePosition);
+});
```

Comment:
```suggestion
const updateDirections = () => {
    updateOpenDirection();
    updateWedgePosition();
};
  
useGlobalEventListener({
  type: "resize",
  listener: () => updateDirections(),
});

// initial update
onMounted(() => updateDirections());
```

We have a utility from other `@sit-onyx/headless` package for this which already covers the unMounted / unmount part :)